### PR TITLE
Install common tools

### PR DESCRIPTION
**Pull request by NicolasMassart**
## PR description
Making Orion doc use the common repository https://github.com/PegaSysEng/doc.common
## Fixed Issue(s)

[PAN-3281](https://pegasys1.atlassian.net/browse/PAN-3281)

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -7,24 +7,30 @@ executors:
   node_executor:
     docker:
       - image: circleci/node:12.9.0-stretch
+  ruby_executor:
+    docker:
+      - image: circleci/ruby:2-stretch
+
```
Comment (joshuafernandes) [inline .circleci/config.yml:14]: Would recommend adding this line to both exectuors
`  working_directory: ~/project
`
And you can remove the './' for the checksum; either way works though :)

File: .circleci/config.yml [hunk 2]
```diff
@@ -35,63 +41,62 @@ jobs:
...
-          key: deps-{{ checksum "package.json" }}
+          key: deps-{{ checksum "./common/build_tools/package.json" }}
       - store_artifacts:
           path: ./linkchecker.out
-          destination: linkchecker.out
+          destination: ./linkchecker.out
 
   markdownlint:
-    executor: python_executor
+    executor: ruby_executor
```
Comment (joshuafernandes) [inline .circleci/config.yml:85]: Very nice :)

**Review by joshuafernandes — APPROVED** [review]
LGTM

File: .circleci/config.yml (hunk 2 above)
Comment (NicolasMassart) [reply .circleci/config.yml:85]: thanks

File: .circleci/config.yml (hunk 1 above)
Comment (NicolasMassart) [reply .circleci/config.yml:14]: for the ./ it was to be consistant among all the file paths as required by @EdJoJob on the EthSigner PR similar to this one.

**Pull request merged**